src/index.html

```diff
@@ -13,6 +13,39 @@
     />
```

Comment:
The `href` attribute is pointing to a `.scss` file. Typically, `.scss` files need to be compiled into `.css` files before they can be used in an HTML document. Ensure that the SASS file is compiled to CSS and the link is updated accordingly.